webapp/components/loading-state.html

```diff
@@ -37,10 +37,14 @@
         }
       }
 
-      async load(promise) {
+      async load(promise, opt_errHandler) {
```

Comment:
maybe a naive question, but should there be a change to calling code to pass in opt_errHandler?

Reply:
Cool thanks for pointing that out.